contracts/contracts/testnet/LynxSet.sol

```diff
@@ -0,0 +1,48 @@
+// SPDX-License-Identifier: AGPL-3.0-or-later
```

Comment:
This file is a great idea 💯

Reply:
Totally agree. This was @vzotova 's suggestion 🎸